src/PerformanceTests/Common/Scenarios/Receive/ReceiveRunner.V5.cs

```diff
@@ -8,19 +9,31 @@ partial class ReceiveRunner : ICreateSeedData
 {
     public int SeedSize { get; set; } = 50000;
     
-    public partial class Handler : IHandleMessages<Command>
+    public partial class Handler : IHandleMessages<Command>, IHandleMessages<EncryptedCommand>
```

Comment:
+1, the baserunner and looprunner are there to help create different scenarios. It's just a few lines of code to create a new one.